fe/fe-core/src/main/java/com/starrocks/alter/LakeTableSchemaChangeJob.java

```diff
@@ -847,6 +852,10 @@ List<MaterializedIndex> visualiseShadowIndex(@NotNull LakeTable table) {
 
                 droppedIndexes.add(droppedIdx);
             }
+
+            long shadowShardGroupId = shardGroupIdMap.get(partition.getId());
```

Comment:
If this job was created before the upgrade, will the `shardGroupIdMap` be null?

Reply:
fixed